### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by Solobarine**
This pull request is to review the following done to the Project.
- Migrations
- Foreign Keys

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Solobarine  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 
--  is missing the Future features Header in the Readme. kindly add this and atleast 1-3 future implementations you will add. this is to make your readme professional
--- please remove all the description comments from your readme (comments meant to guide you in the different sections)
--- please add a good PR summary. it should not be too brief like the one you have

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    t.index ['user_id'], name: 'index_likes_on_user_id'
+  end
+
+  create_table 'posts', force: :cascade do |t|
+    t.string 'title'
+    t.text 'text'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
+    t.bigint 'user_id', null: false
+    t.index ['user_id'], name: 'index_posts_on_user_id'
```
Comment (ichala) [inline db/schema.rb]: - [ ] your post table is missing` :comments_counter ` & `:likes_counter` kindly add them using migration 🙏

image

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  end
+
+  create_table 'users', force: :cascade do |t|
+    t.string 'name'
+    t.text 'photo'
+    t.text 'bio'
+    t.datetime 'created_at', null: false
+    t.datetime 'updated_at', null: false
```
Comment (ichala) [inline db/schema.rb]: - [ ]  your user table is missing ` :posts_counter` kindly add it using migration 🙏

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi Student
Good job so far! 
👏👏👏 
You still need to work on some issues to go to the next project, but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linter errors  ✔️ 
- Used Migration ✔️


 ## Required Changes ♻️

- [ ] **_🛑 x_**
- [ ] **_🛑  Check The Comments Below_**





## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Solobarine , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**